examples/chef/chef.py

```diff
@@ -640,11 +658,12 @@ def main() -> int:
 
         if options.build_target == "esp32":
             shell.run_cmd(f"cd {_CHEF_SCRIPT_PATH}/esp32")
-            if options.do_clean:
-                shell.run_cmd(f"rm -f {_CHEF_SCRIPT_PATH}/esp32/sdkconfig")
-                shell.run_cmd(f"cd {_CHEF_SCRIPT_PATH}/esp32")
-                shell.run_cmd(f"rm -rf {_CHEF_SCRIPT_PATH}/esp32/build")
-                shell.run_cmd("idf.py fullclean")
+            if options.ipv6only:
+                shell.run_cmd(
+                    f"sed -i 's/#\\ CONFIG_DISABLE_IPV4\\ is\\ not\\ set/CONFIG_DISABLE_IPV4=y/g' sdkconfig ")
+            else:
+                shell.run_cmd(
+                    f"sed -i 's/CONFIG_DISABLE_IPV4=y/#\\ CONFIG_DISABLE_IPV4\\ is\\ not\\ set/g' sdkconfig ")
```

Comment:
1. Appending could cause multiple lines appended when we run `chef.py` multiple times.

2. `CONFIG_DISABLE_IPV4=n` is not good because it could be replaced to `#\\ CONFIG_DISABLE_IPV4\\ is\\ not\\ set`  when doing `idf.py menuconfig`